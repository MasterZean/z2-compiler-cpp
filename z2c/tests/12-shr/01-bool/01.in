true >> true
true >> false
true >> 4s8
true >> 4u8
true >> 4s16
true >> 4u16
true >> 4
true >> 4u
true >> 4l
true >> 4ul
true >> 4.0f
true >> 4.0
true >> '4'
true >> 4p
