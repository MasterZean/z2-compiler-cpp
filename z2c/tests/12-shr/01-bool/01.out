DummyClass(1, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Bool'
DummyClass(2, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Bool'
DummyClass(3, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Small'
DummyClass(4, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Byte'
DummyClass(5, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Short'
DummyClass(6, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Word'
DummyClass(7, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Int'
DummyClass(8, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'DWord'
DummyClass(9, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Long'
DummyClass(10, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'QWord'
DummyClass(11, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Float'
DummyClass(12, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Double'
DummyClass(13, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'Char'
DummyClass(14, 6): error:
	can't apply operator '>>' on types: 
		'Bool' and 
		'PtrSize'
